test/test_torch.py

```diff
@@ -14226,6 +14226,23 @@ def test_min_max_binary_op_nan(self, device, dtype):
             self.assertFalse(torch.isnan(ma[i]), "max(a, b): {}, a: {}, b: {}".format(ma[i], a[i], b[i]))
             self.assertFalse(torch.isnan(mi[i]), "min(a, b): {}, a: {}, b: {}".format(mi[i], a[i], b[i]))
 
+    @dtypesIfCUDA(torch.half, torch.float, torch.double)
+    @dtypes(torch.float, torch.double)(
+    def test_min_max_empty_element(self, device, dtype):
+        a = torch.rand(0, dtype=dtype, device=device)
```

Comment:
You don't need to create so many tensors, but you need to vary the dimension with size zero and test tensors with multiple size zero dimensions, too. For example, what happens when you create a tensor with the shape [1, 0], not [0, 1]?